components/bl-knob-component/src/index.js

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+
+import primereact from './lib/core';
+
+const Knob = primereact.knob.Knob;
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function MyCustomComponent({ component, eventHandlers, appData, pageData, parentDataModel }) {
```

Comment:
please, rename it `MyCustomComponent`

can we remove `appData, pageData, parentDataModel ` ?